docs/usage/configuration/members.md

```
@@ -640,7 +638,7 @@ Members within a same category will be ordered according to the [`members_order`
 
 When rendering a namespace, show its subnamespaces recursively.
 
-This is false by default, because most of the time we render only one namespace per page, and when rendering a full package (a tree of namespaces and their members) on a single page, we quickly run out of [heading levels][heading_level].
+This is false by default, because most of the time we render only one namespace per page, and when rendering a full package (a tree of namespaces and their members) on a single page, we quickly run out of [heading levels][#].
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "#"

(MD052, reference-links-images)